README.md

```diff
@@ -1,4 +1,293 @@
+            unified: <cat-allocation-spec>
+      ```
+
+1. `<cache-ids>` may be omitted if no cache id specific configuration (and no
+   CDP config for CAT) is desired, i.e.
+
+      ```
+        l3Allocation: "60%"
+        mbAllocation: ["50%"]
+      ```
+
+   is equal to
+
+      ```
+        l3Allocation:
+          all:
+            unified: "60%"
+        mbAllocation:
+          all: ["50%"]
+      ```
+
+### Examples
+
+Below is a config snippet that would allocate (ca.) 60% of the L3 cache lines
+exclusively to the guaranteed class. The remaining 40% L3 is for burstable and
+besteffort, Besteffort getting only 50% of this. guaranteed class gets full
+memory bandwidth whereas the other classes are throttled to 50%.
+
+```yaml
+options:
+  l2:
+    optional: true
+  l3:
+    optional: true
+  mb:
+    optional: true
+partitions:
+  exclusive:
+    # Allocate 80% of all L2 cache IDs to the "exclusive" partition
+    l2Allocation: "80%"
+    # Allocate 60% of all L3 cache IDs to the "exclusive" partition
+    l3Allocation: "60%"
+    mbAllocation: ["100%"]
+    classes:
+      guaranteed:
+        # Allocate all of the partitions cache lines and memory bandwidth to "guaranteed"
+        l2Allocation: "100%"
+        l3Allocation: "100%"
+        #mbAllocation: ["100%"]
+  shared:
+    # Allocate 20% of L2 and 40% L3 cache IDs to the "shared" partition
+    # These will NOT overlap with the cache lines allocated for "exclusive" partition
+    l2Allocation: "20%"
+    l3Allocation: "40%"
+    mbAllocation: ["50%"]
+    classes:
+      burstable:
+        # Allow "burstable" to use all cache lines of the "shared" partition
+        l2Allocation: "100%"
+        l3Allocation: "100%"
+        #mbAllocation: ["100%"]
+      besteffort:
+        # Allow "besteffort" to use all L2 but only half of the L3 cache
+        # lines of the "shared" partition.
+        # These will overlap with those used by "burstable"
+        l2Allocation: "100%"
+        l3Allocation: "50%"
+        #mbAllocation: ["100%"]
+```
+
+The configuration also supports far more fine-grained control, e.g. per
+cache-ID configuration (i.e. different cahed ids, or sockets, having different
+allocation) and Code and Data Prioritization (CDP) allowing different cache
+allocation for code and data paths.
+
+```yaml
+...
+    partitions:
+      exclusive:
+        l3Allocation: "60%"
+        mbAllocation: ["100%"]
+        classes:
+          # Automatically gets 100% of what was allocated for the partition
+          guaranteed:
+      shared:
+        l3Allocation:
+          # 'all' denotes the default and must be specified
+          all: "40%"
+          # Specific cache allotation for cache-ids 2 and 3
+          2-3: "20%"
+        mbAllocation: ["100%"]
+        classes:
+          burstable:
+            l3Allocation:
+              all:
+                unified: "100%"
+                code: "100%"
+                data: "80%"
+              mbSchema:
```

Comment:
Maybe a word about the difference of mbAllocation and mbSchema would be useful.

Reply:
Woops, it should be `mbAllocation` everywhere (the config interface patch merged earlier renamed that) 😬  Fixed